src/app/core/entity/entity-mapper.service.ts

```diff
@@ -93,6 +122,7 @@ export class EntityMapperService {
     const rawData = this.entitySchemaService.transformEntityToDatabaseFormat(
       entity
     );
+    this.sendUpdate(entity, entity._rev === undefined ? "new" : "update");
```

Comment:
Good find! I did not think about that!
This opens a broader topic thou because the way it is currently implemented is the way it was before - at least when it worked (if I'm not mistaken). Maybe this should then not publish but instead send a notification to the user?